### PR TITLE
Donation Form Contact Info Page (3)

### PR DESCRIPTION
**Pull request by meganniu**
[Figma](https://www.figma.com/file/4EFYFvPUgv1aBdzkE12xn3/Arranging-a-Donation-(Phase-2)?node-id=518%3A2467)
[Design Spec](https://www.notion.so/uwblueprintexecs/72616a24aa4e482b84bf683f919eae0f?v=378202a223194dbcbf8b60847a3f9618&p=ea44a0bc7c3543bf9dffa033344d5b36)

See the component at `http://localhost:3000/test` 

Screenshot from 2021-07-02 21-26-08
Screenshot from 2021-07-02 21-26-17
Screenshot from 2021-07-02 21-27-08

### CODE REVIEW

**Review by kevin-wangg — APPROVED** [review]
LGTM! I ran it locally and verified all the fields worked as expected.

File: client/src/pages/DonationFormContactInfoPage.tsx [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+                    errorString={phoneNumberError}
+                    isDisabled={false}
+                    showErrorIcon={false}
+                    inputComponent={
+                        <TextField
+                            input={phoneNumber}
+                            isDisabled={false}
+                            isErroneous={phoneNumberError.length !== 0}
+                            onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
+                                onPhoneNumberChange(e.target.value);
```
Comment (Puepis) [inline client/src/pages/DonationFormContactInfoPage.tsx:177]: the design specfies to format the phone number as the user types, do we need to implement this or is it too much?

**Review by Puepis — COMMENTED** [review]
LGTM! just made a comment about the phone # formatting

**Review by Puepis — APPROVED** [review]

File: client/src/pages/DonationFormContactInfoPage.tsx (hunk 1 above)
Comment (meganniu) [reply client/src/pages/DonationFormContactInfoPage.tsx:177]: LOL i talked to amanda and we're scrapping that requirement

Comment (meganniu) [reply client/src/pages/DonationFormContactInfoPage.tsx:177]: Sorry, I'll update the design specs rn

**Pull request merged**